metrics/fidelity/fidelity.py

```diff
@@ -0,0 +1,145 @@
+import logging
+
+import numpy as np
+
+import metrics.fidelity.utility as utility
+import datasets
+
+
+logger = logging.getLogger(__name__)
+
+_CITATION = """
+citation is under construction 
+@misc{carton2020evaluating,
+      title={Evaluating and Characterizing Human Rationales}, 
+      author={Samuel Carton and Anirudh Rathore and Chenhao Tan},
+      year={2020},
+      eprint={2010.04736},
+      archivePrefix={arXiv},
+      primaryClass={cs.CL}
+}
+"""
+
+_DESCRIPTION = """
+description is under construction
+sufficiency(x, y_hat, alpha) = 1 - max(0, P(y_hat/x) - P(y_hat/x, alpha))
+comprehensiveness(x, y_hat, alpha) = max(0, P(y_hat/x) - P(y_hat/x, alpha))
+null difference = max(0, P(y_hat/x) - P(y_hat/x, 0))
+"""
+
+_KWARGS_DESCRIPTION = """
+Args:
+	`predictions`: (numpy array) y_hat
+	`prob_y_hat`: (numpy array) P(y_hat/x)
+	`prob_y_hat_alpha`: (numpy array) P(y_hat/x, alpha)
```

Comment:
we should explain in the documentation that alpha is different for computing sufficiency and comprehensivess